<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Catch Keeper - Fishing Log</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Type your catch notes...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Log Catch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter Catch Details</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <property name="indent">
     <number>2</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSpecies">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Large Mouth Bass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Small Mouth Bass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>White Bass</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bluegill</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sunfish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow Perch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Walleye</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Channel Catfish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue Catfish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Flathead Catfish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Northern Pike</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Musky</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rainbow Trout</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Steelhead</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Species:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Weight (lb):</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinWeight">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>65.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Length (in):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinLength">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateCatch">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="timeSpec">
     <enum>Qt::TimeSpec::LocalTime</enum>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeCatch">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>118</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fight Rating (1-10):</string>
    </property>
   </widget>
   <widget class="QSlider" name="sliderFight">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkPB">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Personal Best?</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressCast">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="textHistory">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>301</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial Rounded MT'; font-size:9pt; font-weight:700; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Catch History Appears Here ....&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Weather:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboWeather">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Sunny</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Partly Cloudy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Light Rain</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Overcast</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stormy Conditions</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Foggy</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Progress of Logged Catch ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>721</width>
      <height>90</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>90</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/catchkeeperbar.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
